libsolidity/codegen/ABIFunctions.h

```diff
@@ -134,6 +134,15 @@ class ABIFunctions
 	/// otherwise an assertion failure.
 	std::string cleanupFunction(Type const& _type, bool _revertOnFailure = false);
 
+	/// Performs cleanup after reading from a potentially compressed storage slot.
```

Comment:
Should mention that this expects input data to be right aligned (and it will rectify that).

Reply:
It does so in line 140, doesn't it?

Reply:
Yep, it does!